gatherers/censys.py

```diff
@@ -2,6 +2,7 @@
 import json
 import csv
 import logging
+from typing import List
```

Comment:
Will this lock us to Python 3.7+ as written?

Reply:
No, typing supported this some time back. It definitely works in my local 3.6 environment, and I'm sure it works in 3.5 as well.